code/defines/procs/admin.dm

```diff
@@ -1,7 +1,5 @@
 // Always return "Something/(Something)", even if it's an error message.
 /proc/key_name(whom, include_link = FALSE, type = null)
-	if(length(args) > 3)	// fucked up somewhere, REMOVE THIS BEFORE COMITTING
```

Comment:
I forgot about this lol

Reply:
This is what helped me find where it was broken! I double checked that there wasn't any other calls with four args so I think it should be safe to remove now...